README.md

```diff
@@ -4,9 +4,10 @@ Facilitates migrating data from a CHT 3.x installation to a CHT 4.x installation
 Also, facilitates moving data within a CHT 4.x CouchDb cluster.
 
 #### Installation
+
+Check out this repository. Run:
```

Comment:
I'd feel more comfortable if it was published to our AWS docker repo so we can version it. What do you think?

Reply:
Yea, agree. I'll check if I can create a public ECR repository (otherwise tag site-reliability to create one) and add a publish step to the CI in a different PR.

Reply:
It's ok if publishing is a manual documented step. I don't expect us to release this more than a few times and then it'll be EOL.